clang-tools-extra/docs/clang-tidy/checks/modernize/avoid-variadic-functions.rst

```diff
@@ -0,0 +1,17 @@
+.. title:: clang-tidy - modernize-avoid-variadic-functions
+
+modernize-avoid-variadic-functions
+==================================
+
+Find all function definitions (but not declarations) of C-style variadic
+functions.
+
+Instead of C-style variadic functions, C++ function parameter pack or currying
```

Comment:
As a non-native speaker, I have no idea what `currying` means :) and I cannot find that as an "official" C++ term in cppreference. I would vote for just removing this bit and keeping only parameter packs.